src/main/webapp/homepage.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Sullivan</title>
+    <link rel="stylesheet" type="text/css" href="homepage.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
```

Comment:
What does this line do?

Reply:
I think it's for mobile browser support.

Reply:
Cool! I'll add that to mine too then :)